patches/corefx/0004-Conditionally-include-runtime.native.System.Data.Sql.patch

```diff
@@ -0,0 +1,35 @@
+From 662a705aaf27963558de823a559946d9ee9cf006 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Tue, 1 Oct 2019 14:13:55 +0000
+Subject: [PATCH] Conditionally include
+ runtime.native.System.Data.SqlClient.sni
+
+---
+ external/runtime/runtime.depproj | 4 +++-
+ 1 file changed, 3 insertions(+), 1 deletion(-)
+
+diff --git a/external/runtime/runtime.depproj b/external/runtime/runtime.depproj
+index 3ce0ed8..793eb4e 100644
+--- a/external/runtime/runtime.depproj
++++ b/external/runtime/runtime.depproj
+@@ -19,7 +19,6 @@
+       <Version Condition="'$(TargetFramework)'=='uap10.0.16299'">2.1.0-b-uwp6-25707-02</Version>
+     </PackageReference>
+     <PackageReference Include="Microsoft.NETCore.TestHost" Version="$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)" />
+-    <PackageReference Include="runtime.native.System.Data.SqlClient.sni" Version="$(RuntimeNativeSystemDataSqlClientSniPackageVersion)" />
+     <PackageReference Include="Microsoft.NETCore.DotNetHost" Version="$(MicrosoftNETCoreDotNetHostPackageVersion)" />
+     <PackageReference Include="Microsoft.NETCore.DotNetHostPolicy" Version="$(MicrosoftNETCoreDotNetHostPolicyPackageVersion)" />
+     <!-- We do not need apphost.exe and the 3.0 SDK will actually remove it.
+@@ -27,6 +26,9 @@
+          This can be removed once we have a new SDK -->
+     <FileToExclude Include="apphost" />
+   </ItemGroup>
++  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
++    <PackageReference Include="runtime.native.System.Data.SqlClient.sni" Version="$(RuntimeNativeSystemDataSqlClientSniPackageVersion)" />
```

Comment:
You only needed this here and not here? https://github.com/dotnet/corefx/blob/c22714d65151968a17c3b232748880d5e7961dee/src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.csproj#L40

Are prebuilts permitted for test projects?

Reply:
We're not building the test project in source-build.